backend/legacyapi/setting.go

```diff
@@ -27,6 +27,8 @@ const (
 	SettingPluginOpenAIKey SettingName = "bb.plugin.openai.key"
 	// SettingPluginOpenAIEndpoint is used for OpenAI's API endpoint.
 	SettingPluginOpenAIEndpoint SettingName = "bb.plugin.openai.endpoint"
+	// SettingPluginOpenAIModel is used for OpenAI's API model name
+	SettingPluginOpenAIModel SettingName = "bb.plugin.openai.model"
```

Comment:
Also need to define the constant in /frontend/src/types/setting.ts

Reply:
OK, I will update soon

Reply:
done